apiv2/db_import/tests/test_db_alignment_import.py

```diff
@@ -0,0 +1,146 @@
+    sync_db_session: Session,
+    verify_dataset_import: Callable[[list[str]], models.Dataset],
+    verify_model: Callable[[Base, dict[str, Any]], None],
+    expected_alignments: list[dict[str, Any]],
+) -> None:
+    populate_alignments(sync_db_session)
+    populate_stale_run(sync_db_session)
+    populate_stale_alignments(sync_db_session)
+    sync_db_session.commit()
+    actual = verify_dataset_import(import_alignments=True)
+    expected_iter = iter(expected_alignments)
+    for run in sorted(actual.runs, key=lambda x: x.name):
+        for alignment in run.alignments:
+            expected = next(expected_iter)
+            if "run_id" not in expected:
+                expected["run_id"] = run.id
+            verify_model(alignment, expected)
+
+
+def test_import_per_section_alignment_parameters(
+    sync_db_session: Session,
+    verify_dataset_import: Callable[[list[str]], models.Dataset],
+    verify_model: Callable[[Base, dict[str, Any]], None],
+    expected_per_section_alignment_parameters: list[dict[str, Any]],
+) -> None:
+    populate_per_section_alignment_parameters(sync_db_session)
+    sync_db_session.commit()
+    actual = verify_dataset_import(import_alignments=True)
+    expected_per_section_alignments_iter = iter(expected_per_section_alignment_parameters)
+
+    for run in sorted(actual.runs, key=lambda x: x.name):
+        for tiltseries in run.tiltseries:
+            for alignment in tiltseries.alignments:
+                for per_section_alignment in alignment.per_section_alignments:
+                    expected = next(expected_per_section_alignments_iter)
+                    if "alignment_id" not in expected:
+                        expected["alignment_id"] = alignment.id
+                    verify_model(per_section_alignment, expected)
+
+
+def test_import_per_section_alignment_parameters_stale_deletion(
+    sync_db_session: Session,
+    verify_dataset_import: Callable[[list[str]], models.Dataset],
+    verify_model: Callable[[Base, dict[str, Any]], None],
+    expected_per_section_alignment_parameters: list[dict[str, Any]],
+) -> None:
+    populate_per_section_alignment_parameters(sync_db_session)
+    populate_stale_run(sync_db_session)
+    populate_stale_per_section_alignment_parameters(sync_db_session)
+    sync_db_session.commit()
+    actual = verify_dataset_import(import_alignments=True)
+    expected_per_section_alignments_iter = iter(expected_per_section_alignment_parameters)
+    for run in sorted(actual.runs, key=lambda x: x.name):
+        for tiltseries in run.tiltseries:
+            for alignment in tiltseries.alignments:
+                for per_section_alignment in alignment.per_section_alignments:
+                    expected = next(expected_per_section_alignments_iter)
+                    if "alignment_id" not in expected:
+                        expected["alignment_id"] = alignment.id
+                    verify_model(per_section_alignment, expected)
```

Comment:
It would be nice to add an extra assertion to these tests to check that we called `verify_model` the expected number of times, in order to ensure that we don't wind up skipping any iterations and silently ignoring failure scenarios